apps/prairielearn/elements/pl-number-input/pl-number-input.mustache

```diff
@@ -12,26 +12,53 @@
 {{#inline}}<span class="form-inline d-inline-block ml-2">{{/inline}}
     <span id="pl-number-input-{{uuid}}" class="input-group pl-number-input">
         {{#label}}
+        <label for="pl-number-input-textfield-{{uuid}}" hidden>{{accessibility_description}}</label>
         <span class="input-group-prepend">
-            <span class="input-group-text">{{{label}}}</span>
+            <span class="input-group-text" id="pl-number-input-{{uuid}}-label">{{{label}}}</span>
         </span>
         {{/label}}
-        <input name="{{name}}" autocomplete="off" type="text" class="form-control {{^display_append_span}}rounded-right{{/display_append_span}} pl-number-input-input" size="{{size}}" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
+        <input
+          id="pl-number-input-textfield-{{uuid}}" 
+          name="{{name}}"
+          type="text" 
+          inputmode="text"
+          autocomplete="off" 
+          class="form-control pl-number-input-input {{^display_append_span}}rounded-right{{/display_append_span}}" 
+          size="{{size}}" 
+          {{^editable}}disabled{{/editable}} 
+          {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} 
+          aria-describedby="pl-integer-input-{{uuid}}-label pl-number-input-{{uuid}}-suffix" 
+          placeholder="{{shortinfo}}" />
 
           <span class="input-group-append">
-            {{#suffix}}<span class="input-group-text {{^show_info}}rounded-right{{/show_info}}">{{suffix}}</span>{{/suffix}}
+            {{#suffix}}
+            <span class="input-group-text {{^show_info}}rounded-right{{/show_info}}" id="pl-number-input-{{uuid}}-suffix">{{suffix}}</span>
+            {{/suffix}}
             {{#show_info}}
               <a role="button" class="btn btn-light border d-flex align-items-center" data-toggle="popover" data-html="true" title="{{popup_title}}" data-content="{{info}}" data-placement="auto" data-trigger="focus" tabindex="0">
               <i class="fa fa-question-circle" aria-hidden="true"></i>
             {{/show_info}}
             {{^show_info}}
               <span class="d-flex align-items-center">
             {{/show_info}}
-
-            {{#correct}}&nbsp;<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
-            {{#partial}}&nbsp;<span class="badge badge-warning"><i class="fa fa-circle-o" aria-hidden="true"></i> {{partial}}%</span>{{/partial}}
-            {{#incorrect}}&nbsp;<span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>{{/incorrect}}
-
+            {{#parse_error}}
+                 <a role="button"
+                   class="btn btn-light border d-flex align-items-center text-danger rounded-right"
```

Comment:
As I have now discovered, we can just get rid of all the rounded rights: https://github.com/PrairieLearn/PrairieLearn/pull/8270

Reply:
Removed as suggested.